Fastlane/testing_lanes.rb

```diff
@@ -56,18 +61,18 @@
       output_types: '',
       # xcodebuild_formatter: '', # Add this to get verbose logging by disabling xcbeautify.
       suppress_xcode_output: false,
-      buildlog_path: ENV['BITRISE_DEPLOY_DIR'],
+      buildlog_path: ENV['BITRISE_DEPLOY_DIR'], # Makes sure our build log is deployed and available in Bitrise.
```

Comment:
Not sure if this comment is accurate. Isn't it setting the build log path? How would it make sure it's deployed and available in Bitrise?

Reply:
Ah, I should be more explicit. The fact that we use `BITRISE_DEPLOY_DIR` makes sure we deploy it. I'll update!